components/org.wso2.carbon.stream.processor.core/src/main/java/org/wso2/carbon/stream/processor/core/persistence/beans/PersistenceConfigurations.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.stream.processor.core.persistence.beans;
+
+import org.wso2.carbon.config.annotation.Configuration;
+
+@Configuration(namespace = "state.persistence", description = "Query configurations for state persistence")
+public class PersistenceConfigurations {
+    private boolean enabled = false;
+    private int intervalInMin = 1;
+    private int revisionsToKeep = 3;
+    private String persistenceStore = "org.wso2.carbon.stream.processor.core.persistence.FileSystemPersistenceStore";
```

Comment:
Better to use FileSystemPersistenceStore.class.getName()

Reply:
Fixed in commit 8d13bdc